src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -191,197 +328,93 @@ private ExecutorInfo createExecutor(String taskIdName, String nodeName, String e
             .build());
   }
 
-  private void launchDataNode(SchedulerDriver driver, Offer offer) {
-    launchNode(
-        driver,
-        offer,
-        HDFSConstants.DATA_NODE_ID,
-        Arrays.asList(HDFSConstants.DATA_NODE_ID),
-        HDFSConstants.NODE_EXECUTOR_ID);
-  }
-
-  private void launchInitialJournalNodes(SchedulerDriver driver, Collection<Offer> offers) {
-    for (int i = 0; i < conf.getJournalNodeCount(); i++) {
-      if (offers.size() > 0) {
-        Offer offer = offers.iterator().next();
-        pendingOffers.remove(offer.getId());
-        launchNode(
-            driver,
-            offer,
-            HDFSConstants.JOURNAL_NODE_ID,
-            Arrays.asList(HDFSConstants.JOURNAL_NODE_ID),
-            HDFSConstants.NODE_EXECUTOR_ID);
-      }
+  private boolean maybeLaunchJournalNode(SchedulerDriver driver, Offer offer) {
```

Comment:
How about "couldLaunchJournalNode", since maybe sounds all kinds of doubtful to me? "Could" is a solid boolean.